### ISSUE TITLE
Packer 1.4.0 crashed while configuring VNC port for remote vmware-iso build

### ISSUE THREAD

**Issue by BeverlyBrown** [hashicorp/packer#7522]
Using a json template that worked fine with Packer version 1.3.5, I am now seeing it crash when configuring the VNC port . Output below - I attached the crash.log file.

==> Builds finished but no artifacts were created.
panic: runtime error: invalid memory address or nil pointer dereference
2019/04/17 17:40:49 packer: [signal SIGSEGV: segmentation violation code=0x1 addr=0x10 pc=0x1df735c]
2019/04/17 17:40:49 packer: 
2019/04/17 17:40:49 packer: goroutine 52 [running]:
2019/04/17 17:40:49 packer: github.com/hashicorp/packer/builder/vmware/common.(*StepConfigureVNC).Run(0xc0000566c0, 0x329bca0, 0xc00031cf40, 0x3286aa0, 0xc00074a060, 0x0)
2019/04/17 17:40:49 packer:     /Users/mmarsh/go/src/github.com/hashicorp/packer/builder/vmware/common/step_configure_vnc.go:120 +0x7dc
2019/04/17 17:40:49 packer: github.com/hashicorp/packer/helper/multistep.(*BasicRunner).Run(0xc00074a1e0, 0x329bca0, 0xc00031cf40, 0x3286aa0, 0xc00074a060)
2019/04/17 17:40:49 packer:     /Users/mmarsh/go/src/github.com/hashicorp/packer/helper/multistep/basic_runner.go:67 +0x1ec
2019/04/17 17:40:49 packer: github.com/hashicorp/packer/builder/vmware/iso.(*Builder).Run(0xc000382e00, 0x329bca0, 0xc00031cf40, 0x32abda0, 0xc00067a1c0, 0x324c860, 0xc000250070, 0x40b8b9, 0xc00032fce0, 0x30, ...)
2019/04/17 17:40:49 packer:     /Users/mmarsh/go/src/github.com/hashicorp/packer/builder/vmware/iso/builder.go:183 +0x1488
2019/04/17 17:40:49 packer: github.com/hashicorp/packer/packer/rpc.(*BuilderServer).Run(0xc0002fc510, 0x1, 0xc000678070, 0x0, 0x0)
2019/04/17 17:40:49 packer:     /Users/mmarsh/go/src/github.com/hashicorp/packer/packer/rpc/builder.go:109 +0x231
2019/04/17 17:40:49 packer: reflect.Value.call(0xc0002845a0, 0xc0003440f0, 0x13, 0x2d81824, 0x4, 0xc0000c9f18, 0x3, 0x3, 0xc00002a000, 0x3, ...)
2019/04/17 17:40:49 packer:     /usr/local/go/src/reflect/value.go:447 +0x461
2019/04/17 17:40:49 packer: reflect.Value.Call(0xc0002845a0, 0xc0003440f0, 0x13, 0xc00071e718, 0x3, 0x3, 0xc0003e0a20, 0xc00071e730, 0x404e35)
2019/04/17 17:40:49 packer:     /usr/local/go/src/reflect/value.go:308 +0xa4
2019/04/17 17:40:49 packer: net/rpc.(*service).call(0xc000348040, 0xc000480050, 0xc0003f8058, 0xc0003f8060, 0xc0003d8100, 0xc000668280, 0x258b4c0, 0xc00067806c, 0x18a, 0x2506460, ...)
2019/04/17 17:40:49 packer:     /usr/local/go/src/net/rpc/server.go:384 +0x14e
2019/04/17 17:40:49 packer: created by net/rpc.(*Server).ServeCodec
2019/04/17 17:40:49 packer:     /usr/local/go/src/net/rpc/server.go:481 +0x440
2019/04/17 17:40:49 [INFO] (telemetry) ending vmware-iso
2019/04/17 17:40:49 ui error: Build 'vmware-iso' errored: unexpected EOF
2019/04/17 17:40:49 /auto/svn/bbrown/builds/esx70/ftesx-trunk/ftl_linux/appliance/packer: plugin process exited
2019/04/17 17:40:49 Builds completed. Waiting on interrupt barrier...
2019/04/17 17:40:49 machine readable: error-count []string{"1"}
2019/04/17 17:40:49 ui error: 
==> Some builds didn't complete successfully and had errors:
2019/04/17 17:40:49 machine readable: vmware-iso,error []string{"unexpected EOF"}
2019/04/17 17:40:49 ui error: --> vmware-iso: unexpected EOF
2019/04/17 17:40:49 ui: 
==> Builds finished but no artifacts were created.
2019/04/17 17:40:49 [INFO] (telemetry) Finalizing.
2019/04/17 17:40:50 waiting for all plugin processes to complete...
2019/04/17 17:40:50 /auto/svn/bbrown/builds/esx70/ftesx-trunk/ftl_linux/appliance/packer: plugin process exited
2019/04/17 17:40:50 /auto/svn/bbrown/builds/esx70/ftesx-trunk/ftl_linux/appliance/packer: plugin process exited
2019/04/17 17:40:50 /auto/svn/bbrown/builds/esx70/ftesx-trunk/ftl_linux/appliance/packer: plugin process exited
2019/04/17 17:40:50 /auto/svn/bbrown/builds/esx70/ftesx-trunk/ftl_linux/appliance/packer: plugin process exited
2019/04/17 17:40:50 /auto/svn/bbrown/builds/esx70/ftesx-trunk/ftl_linux/appliance/packer: plugin process exited

[crash.log](https://github.com/hashicorp/packer/files/3091700/crash.log)

**Comment by BeverlyBrown** [comment]
My VNC port min and max are:

      "vnc_port_min": "5901",
      "vnc_port_max": "5920"

**Comment by SwampDragons** [comment]
Hi, thanks for reaching out. I'm having trouble reproducing this. Can you share full example template that reproduces this error?

**Comment by SwampDragons** [comment]
Oh, actually I think this is a duplicate of https://github.com/hashicorp/packer/issues/7505 and fixed by https://github.com/hashicorp/packer/pull/7509. 

I'll upload a linux binary to that PR so you can test it out.

**Comment by SwampDragons** [comment]
does this binary work for you? 
[packer.zip](https://github.com/hashicorp/packer/files/3091775/packer.zip)

**Comment by BeverlyBrown** [comment]
Yes, this seems to be working. It's still buiding the VM, but it got past the point where it had crashed.
Thank you for such a quick response!

**Comment by SwampDragons** [comment]
No problem! Let us know if you find anything else :)
